apps/chameleon/components/user-auth-form.tsx

```diff
@@ -80,27 +83,33 @@ export function UserAuthForm({ className, ...props }: UserAuthFormProps) {
               </p>
             )}
           </div>
-          <button className={cn(buttonVariants())} disabled={isLoading}>
+
+          <button
+            className={cn(buttonVariants(variantType))}
+            disabled={isLoading}
+          >
             {isLoading && (
               <Icons.spinner className="mr-2 h-4 w-4 animate-spin" />
             )}
             Sign In with Email
           </button>
         </div>
       </form>
+
       <div className="relative">
         <div className="absolute inset-0 flex items-center">
-          <span className="w-full border-t" />
+          {/* <span className="w-full border-t" /> */}
         </div>
-        <div className="relative flex justify-center text-xs uppercase">
-          <span className="bg-background px-2 text-muted-foreground">
-            Or continue with
-          </span>
+        <div className="relative flex justify-center text-xs font-bold uppercase items-center">
+          <span className="w-[30%] border-t"></span>
```

Comment:
dont use arb values if they can be avoided I am pretty sure there is a tailwind class for this